torch/csrc/jit/codegen/cuda/lower_utils.h

```diff
@@ -75,6 +79,55 @@ const TensorView* asConstTV(const Val* const);
 
 bool isUnrolledFor(const Expr*);
 
+/*
+  Represents mapping to bool from BIDx, BIDy, BIDz, TIDx, TIDy and TIDz.
+ */
+class ParallelTypeBitmap {
+ public:
+  static constexpr int num_p_type = 6;
```

Comment:
does num_p_type have to be public?

Reply:
Currently, yes. It is referenced at https://github.com/naoyam/pytorch/blob/block_broadcast/torch/csrc/jit/codegen/cuda/lower_thread_predicate.cpp#L85. 

It'd be actually nice if we could hide the variable.